sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 # Release History
 
-## 12.21.0b1 (Unreleased)
+## 12.21.0b1 (2024-12-10)
 
 ### Features Added
+- Added support for service version 2025-05-05.
+- Added support for `ShareFileClient` using the `NFS` protocol to `create_hard_link` 
+with an existing file by specifying full path starting from the root.
```

Comment:
Possible refactor ideas: Added new `create_hard_link` method to `ShareFileClient` to create hard links to specified files and is only supported for the `NFS` protocol.

I think this makes it more obvious that it is NFS-only and what the API does rather than how to use the API?